distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email.enso

```diff
@@ -0,0 +1,78 @@
+import project.Data.Text.Text
+import project.Data.Vector.Vector
+import project.Errors.Common.Missing_Argument
+import project.Errors.Common.Dry_Run_Operation
+import project.Network.Email.Email_Address.Email_Address
+import project.Network.Email.Email_Attachment.Email_Attachment
+import project.Network.Email.Email_Body.Email_Body
+import project.Network.Email.Email
+import project.Network.Email.Email_Provider.Email_Provider
+import project.Network.Email.Email_Send_Response.Send_Response
+import project.Runtime.Context
+import project.Warning.Warning
+
+## An email object representing all parts of an email.
+type Email
+    ## An email object with all standard fields.
+    Message from_address:Email_Address to:(Vector Email_Address)=[] cc:(Vector Email_Address)=[] bcc:(Vector Email_Address)=[] subject:Text="" body:Email_Body="" attachments:(Vector Email_Attachment)=[]
+
+    ## ---
+       aliases: [mail]
+       group: Web
+       icon: email
+       ---
+       Creates an email object.
+    @from Email_Address.default_widget
+    @to Email_Address.vector_widget
+    @cc Email_Address.vector_widget
+    @bcc Email_Address.vector_widget
+    @attachments Email_Attachment.vector_widget
+    new from:Email_Address=(Missing_Argument.throw "from") to:(Vector Email_Address)=[] cc:(Vector Email_Address)=[] bcc:(Vector Email_Address)=[] subject:Text="" body:Email_Body="" attachments:(Vector Email_Attachment)=[] -> Email =
+        Email.Message from to cc bcc subject body attachments
+
+    ## ---
+       aliases: [mail]
+       group: Web
+       icon: email
+       ---
+       Sends an email.
+
+       Sends an email using the specified email provider. Only sends an email in write mode.
+       The Test_Only email provider can be used to simulate sending an email without actually sending it.
+    @from Email_Address.default_widget
+    @to Email_Address.vector_widget
+    @cc Email_Address.vector_widget
+    @bcc Email_Address.vector_widget
+    @attachments Email_Attachment.vector_widget
+    send from:Email_Address=(Missing_Argument.throw "from") to:(Vector Email_Address)=[] cc:(Vector Email_Address)=[] bcc:(Vector Email_Address)=[] subject:Text="" body:Email_Body="" attachments:(Vector Email_Attachment)=[] provider:Email_Provider=(Missing_Argument.throw "provider") -> Send_Response =
```

Comment:
- if [users of Enso search for actions](https://github.com/enso-org/enso/pull/14258#discussion_r2513998232)
- then we do we have another `send` method
- having two ways to do the same thing doesn't help [clarity](https://wiki.apidesign.org/wiki/Clarity)

Reply:
- can you justify there are two ways to send an email?
- e.g. are there different use-cases?
   - one for `Email.send ...` 
   - and another one for `Email.new .... . send_message`
- once you said:

> ... users are searching for actions ...

- if that's true, why not have `Email.compose ... . send`?
  - users would _compose an email_ - that's the action
  - and then they could send it 
- that covers all the use-cases I can imagine and it _"feels actionable"_ to me

Reply:
@jdunkerley I think you suggested adding send_message. I would be happy to delete it for now citing YAGNI.

I think the primary method for sending an email should be a single component email.send